src/constants/default-settings.js

```diff
@@ -777,3 +777,7 @@ export const DATASET_FORMATS = keyMirror({
   csv: null,
   keplergl: null
 });
+
+export const DEFAULT_ELEVATION = 500;
```

Comment:
is this really neccessary? I would rather each layer decide what their defaults should be instead of impose a kepler wide value. It also makes it hard to package layers into submodules and only import the ones they need in the future